### PR TITLE
Improve Consistency Documentation

### PR DESCRIPTION
**Pull request by moritzraho**
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ Note that atomicity is ensured, i.e.  `a.get('food')` will never return somethin
 
 ### Adobe I/O Runtime considerations
 
-State lib is expected to be used in Adobe I/O Runtime serverless actions. A new instance is expected to be created on every new invocation inside the main function of the serverless action as follows:
+State lib is expected to be used in Adobe I/O Runtime serverless actions. A new instance cans be created on every new invocation inside the main function of the serverless action as follows:
```
Comment (meryllblanchet) [inline README.md:97]: Typo `cans` -> `can`

File: README.md [hunk 2]
```diff
@@ -104,9 +104,9 @@ function main (params) {
   // do operations on state
 ```
 
-It's important to understand that on every invocation a new State instance is created, meaning that operations will be only **eventually consistent** across invocations but **strongly consistent** within an invocation.
+It's important to understand that in this cas, on every invocation a new State instance is created, meaning that operations will be only **eventually consistent** across invocations but **strongly consistent** within an invocation.
```
Comment (meryllblanchet) [inline README.md:107]: Typo `in this cas,` ->  `in this case,`

**Review by meryllblanchet — CHANGES_REQUESTED** [review]
Thanks @moritzraho ! LGTM, there are two minor typos for which I request a change.

**Review by moritzraho — COMMENTED** [review]
Thanks @meryllblanchet for the review, merging now

**Pull request merged**